app/controllers/users_controller.rb

```diff
@@ -3,4 +3,22 @@ class UsersController < ApplicationController
   def show
     @user = current_user
   end
+
+  def edit
+    @user = User.find(current_user.id)
+  end
+
+  def create
```

Comment:
更新なのでupdateのほうが良さそうです
```suggestion
  def update
```